File: Modul 2/Soal 4/PRAK204-2410817210019-AchmadReihanAlfaiz.py
PI = 22.0 / 7.0

numbers = []

while len(numbers) < 2:
    line = input().strip()
    numbers.extend(map(float, line.split()))

r, t = numbers[:2]

volume = PI * r * r * t
luas = 2 * PI * r * (r + t)
keliling = 2 * PI * r

print(f"\nVolume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul 2/Soal 4/PRAK204-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>
#define PI 22.0 / 7.0

int main()
{
     float r, t;
     scanf("%f %f", &r, &t);

     float volume = PI * r * r * t;
     float luas = 2 * PI * r * (r + t);
     float keliling = 2 * PI * r;

     printf("\nVolume = %.2f\n", volume);
     printf("Luas = %.2f\n", luas);
     printf("Keliling = %.2f\n", keliling);

     return 0;
}